terraform/modules/kubernetes/availability.tf

```diff
@@ -0,0 +1,16 @@
+resource "kubernetes_pod_disruption_budget_v1" "pdb" {
+  count = var.pdb_min_available != null ? 1 : 0
+
+  metadata {
+    name      = "${local.webapp_name}-pdb"
+    namespace = var.namespace
+  }
+  spec {
+    min_available = try(var.pdb_min_available, null)
```

Comment:
Given the condition, `var.pdb_min_available` will never be null. So we could remove the try()